### DISCUSSION TITLE
upload Minifork wifi error status 1

### DISCUSSION THREAD

**Discussion by Lati1185** [espressif/arduino-esp32#10747] [Question - Community Help]
i want to upload a sketch witch i got vom Git hub for mini Fork lift by Professor Boots.
i got the SESP 32 Dev Mode an the connection to the port 
but i got a error 
it´this 

C:/Users/admin/AppData/Local/Arduino15/packages/esp32/tools/esp-x32/2405/bin/../lib/gcc/xtensa-esp-elf/13.2.0/../../../../xtensa-esp-elf/bin/ld.exe: C:\Users\admin\AppData\Local\arduino\sketches\59583E730A7519953D3549F5ECF3CCCB\libraries\ESPAsyncWebSrv\AsyncWebSocket.cpp.o:(.literal._ZN17AsyncWebLockGuardC5ERK12AsyncWebLock[_ZN17AsyncWebLockGuardC5ERK12AsyncWebLock]+0x0): undefined reference to `pxCurrentTCB'
collect2.exe: error: ld returned 1 exit status

exit status 1

Compilation error: exit status 1

how can i solve this

**Comment by jane-pang** [comment]
I had that problem too. This saves my life: https://github.com/me-no-dev/ESPAsyncWebServer/issues/1442
